app/controllers/instructeurs/dossiers_controller.rb

```diff
@@ -56,6 +56,7 @@ def send_to_instructeurs
       recipients = Instructeur.find(params[:recipients])
 
       recipients.each do |recipient|
+        recipient.followed_dossiers << dossier
```

Comment:
J'ai l'impression que le code de `Instructeur#follow` fait un peu de gestion d'erreur que l'accès direct ne fait pas.

Par exemple il me semble que celle ligne va casser si l'instructeur suit déjà le dossier.

```suggestion
        recipient.follow(dossier)
```

Reply:
Bien  vu @kemenaran je fix.